functions/server_array_to_hash.pp

```diff
@@ -0,0 +1,12 @@
+# @summary Function to normalise servers/pools/peers
+#
+# @api private
+function chrony::server_array_to_hash(Variant[Hash,Array] $servers, $options = []) >> Hash {
+  if $servers.is_a(Hash) {
+    $servers
+  } else {
+    $servers.reduce({}) |$memo, $server| { # lint:ignore:manifest_whitespace_opening_brace_before
```

Comment:
I've added a comment to https://github.com/kuleuven/puppet-lint-manifest_whitespace-check/issues/8